integration/__tests__/app.tests.ts

```diff
@@ -3,21 +3,22 @@ import * as utils from "../helpers/utils";
 import { listHelmRepositories } from "../helpers/utils";
 import { fail } from "assert";
 
-
 jest.setTimeout(60000);
 
 // FIXME (!): improve / simplify all css-selectors + use [data-test-id="some-id"] (already used in some tests below)
 describe("Lens integration tests", () => {
   let app: Application;
 
   describe("app start", () => {
-    beforeAll(async () => app = await utils.appStart(), 20000);
+    beforeAll(utils.wrapJestLifecycle(async () => {
```

Comment:
You could reduce duplication with `beforeAllWrapped(fn : () => Promise<void>, ...)`
and `afterAllTearDown()`, afterEachTearDown()`

Reply:
Good idea